src/database/migrations/20240122110653-create-accounts.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.createTable('Accounts', {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      createdAt: {
```

Comment:
for database columns, I'd suggest to sticking with `snake_cased` column names otherwise they become a pain to deal with when querying directly using raw queriers.